vuepress/docs/next/docs/curate/bundle-comparison.md

```diff
@@ -4,7 +4,7 @@ sidebarDepth: 2
 
 # Bundle Evolution on Entando 7.1
 
-Entando 7.1 implements a more modular approach to building apps with the **ent bundle** CLI orchestrating the management of project bundles. The Entando Bundle was restructured, its processes decoupled by component type, and the publishing transport method, streamlined. With these changes, Entando Bundles went from git-based to docker-based OCI bundles. Both methods are supported on Entando 7.1, and this page clarifies the differences between the two, along with the improvements in the new release.
+Entando 7.1 implements a more modular approach to building apps with the **ent bundle CLI** orchestrating the management of project bundles. The Entando Bundle was restructured, its processes decoupled by component type, and the publishing transport method was streamlined. With these changes, Entando Bundles went from git-based to docker-based OCI bundles. Both types are supported on Entando 7.1, and this page clarifies the differences between the two, along with the improvements in the new release.
```

Comment:
just a note: both methods and types are work in this sentence. But it's not just types that are supported in the specific sense, because the way the bundles are processed in the building, packing, etc. is different. The way it is processed is different too so methods is technically applicable too.

Reply:
the subject is bundles, of which there are types. a bundle is not a method. we would need to rephrase / change the subject to make "methods" work

Reply:
the paragraph talks about bundles and how they are processed

Reply:
methods doesn't work here. if anything, it would refer back to the transport method mentioned 2 sentences prior. more directly, the previous sentence concerns bundles. to use a synonym for the subject that refers to something other than a bundle in the next breath doesn't work.

Reply:
You have a point, with the previous sentence, it flows nicer. But the topic sentence of the paragraph is not bundles but how bundles are managed by the ent bundle cli. So both words work, 'type' flows better, while 'method' emphasizes the process.